src/io/flutter/FlutterBundle.properties

```diff
@@ -134,19 +134,19 @@ module.wizard.app_title=Flutter Application
 module.wizard.app_description=Creates a Flutter application
 module.wizard.app_step_title=New Flutter Application
 module.wizard.app_step_body=Configure the new Flutter application
-module.wizard.app_name=awesome_app
+module.wizard.app_name=flutter_app
```

Comment:
`sample_app`? `my_app`?

Reply:
@mit-mit suggested flutter_app. If you two agree it should be changed again then I'll be happy to change it :) For now, I'll merge this.